app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,45 +1,47 @@
 .navbar {
   display:inline-flex;
-  background: $cream;
+  background: $bg_color_white;
   width: 100vw;
-  height: 5vh;
-  padding: 20px;
-}
-
-.form-control {
-margin-top: 12px;
-justify-content: center;
-width: 40vw;
-border-radius: 15px;
-}
-
-.form-search {
-width:20vw;
-}
-
-.form-group {
-  display: inline-flex;
-  height: 0;
 }
 
 .button {
-margin-top: 12px;
-margin-left: none;
-padding:none;
-border: none;
-width: 5vw;
-border-radius: 15px;
-height: 5vh;
+  margin-top: 5px;
+  margin-left: none;
+  padding: none;
+  border: none;
+  width: 5px;
+  border-radius: 15px;
+  height: inherit;
 }
 
 .navbar_avatar {
-  justify-content: space-between;
-  background: $cream;
-  width: 10vw;
-  height: 15vh;
+  background: $bg_color_white;
+  width: 50px;
+  height: auto;
 }
 
 .navbar_avatar .navbar_avatar-profile img {
-  width: 60px;
+  width: 50px;
   border-radius: 90px;
 }
+
+.form-control {
```

Comment:
you can't use this class name as it is already used by bootstrap. Just modify the class name please!